_docs/best-practices.md

```diff
@@ -117,7 +117,17 @@ $ dockstore tool launch --local-entry  metadata_example.cwl --json sample.json
 
 ## Metadata and Authorship
 
-For all developers, we recommend the following minimal metadata for your tool and workflows.
+Dockstore parses metadata and displays it in the tool/workflow's "Info" tab.  Metadata must be present in the descriptor or else the highlighted sections below will not appear.
```

Comment:
Maybe reverse this to be more positive. 
As in, you should explain why its nice to provide metadata, what they get out of it, and then explain how to do it. Remember, this is additional work so some cheerleading/persuasion needs to be done here.